src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrentInfo.cs

```diff
@@ -20,46 +21,35 @@ public static IEnumerable<object[]> CurrentInfo_CustomCulture_TestData()
         [MemberData(nameof(CurrentInfo_CustomCulture_TestData))]
         public void CurrentInfo_CustomCulture(CultureInfo newCurrentCulture)
         {
-            CultureInfo oldCurrentCulture = CultureInfo.CurrentCulture;
-            try
+            RemoteInvoke((cultureCode) =>
             {
-                CultureInfo.CurrentCulture = newCurrentCulture;
-                Assert.Same(newCurrentCulture.NumberFormat, NumberFormatInfo.CurrentInfo);
-            }
-            finally
-            {
-                CultureInfo.CurrentCulture = oldCurrentCulture;
-            }
+                CultureInfo newCulture = new CultureInfo(cultureCode);
+                CultureInfo.CurrentCulture = newCulture;
+                Assert.Same(newCulture.NumberFormat, NumberFormatInfo.CurrentInfo);
+                return SuccessExitCode;
+            }, newCurrentCulture.TwoLetterISOLanguageName).Dispose();
```

Comment:
>I assume your local machine is Windows 10, not Windows 8? 

Yes, it is.

>Why are you using TwoLetterISOLanguageName rather than Name?

I wasn't sure how else to pass the Invariant culture to the RemoteInvokation since it needed a string input. I tried a few different things and stopped when one (TwoLetterISOLanguageName) worked. The empty string was being whitespace-parsed away by the RemoteInvoke call. I originally had it as a local method variable, but that failed for a reason I don't recall. I'll try shifting things around again.